models/club.ts

```diff
@@ -0,0 +1,11 @@
+import { DocumentReference, DocumentData } from "firebase/firestore"
+import Contacts from "./contacts"
+
+interface Club {
+    name: string
+    events?: DocumentReference<DocumentData>[] //references to other events,
```

Comment:
Is this comment partially cut off? It may have not come through correctly.

Reply:
I may need to revise the club datastructure first